### ISSUE TITLE
LNK1104 cannot open file D:/dev/vcpkg/installed/x64-windows/lib/Qt5Widgetsd.lib

### ISSUE THREAD

**Issue by karlfroldan** [microsoft/vcpkg#25307]
**Describe the bug**
Creating a new Qt5 project using Qt Tools for Visual Studio 2022, with Qt5 installed using vcpkg, fails to link with Qt5Wigetsd.lib

**Environment**
- OS: Windows 11 x64
- Compiler: MSVC 2019 x64

**To Reproduce**
1. vcpkg install qt5 --triplet x64-windows
2. Install Qt tools for Visual Studio 2022
3. Add `qmake.exe` from `D:\dev\vcpkg\installed\x64-windows\tools\qt5\bin\qmake.exe` to available Qt versions.
4. Create a new *Qt Widgets Application*
5. Build and run.

**Expected behavior**
The same setup works fine with Qt Creator (by using vcpkg-installed Qt5 instead of using Qt installer). Visual Studio should compile the Qt widgets application too.

Also, it seems `Qt5Widgetsd.lib` should be in the `lib` folder but it's missing. Only `Qt5Windgets.lib` is there.

**Failure logs**
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	LNK1104	cannot open file 'D:\dev\vcpkg\installed\x64-windows\lib\Qt5Widgetsd.lib'	QtWidgetsApplication1	D:\projects\QtWidgetsApplication1\LINK	1	

```

**Comment by JackBoosY** [comment]
Can you please close your anti-virus then try again?

Thanks.

**Comment by karlfroldan** [comment]
I disabled Windows Defender and reinstalled qt5. The library is still missing. The library is also missing from `x64-linux`. 

Since it works when using Qt Creator, could this be a problem with VS Qt Tools instead of vcpkg?

**Comment by JackBoosY** [comment]
Maybe.
I'm not the professional.
cc @Neumann-A

**Comment by Neumann-A** [comment]
> Qt Tools for Visual Studio

the tools are stupid. Please feel free to complain upstream about not simply accepting a `qt.conf` for different configurations (debug/release) but requiring an explicit `qmake.exe` binary in a bin directory.

`D:\dev\vcpkg\installed\x64-windows\tools\qt5\bin\qmake.exe` will only work with release.

for debug use
`D:\dev\vcpkg\installed\x64-windows\tools\qt5\debug\bin\qmake.exe` (or where ever i put the debug version; needs to have `/debug` in the path)

Basically you need to feed qmake and all the other tools a different `qt.conf`

**Comment by karlfroldan** [comment]
> for debug use `D:\dev\vcpkg\installed\x64-windows\tools\qt5\debug\bin\qmake.exe` 
this solved the issue. I didn't see the `debug` folder earlier. Thanks a lot!

**Comment by connectioncreamy** [comment]
> Maybe. I'm not the professional. cc [@Neumann-A](https://github.com/Neumann-A)





https://wysihtml5-starry.github.io/lisandro/
